dev/publish_release.sh

```diff
@@ -92,13 +93,27 @@ function publish_to_maven() {
 
   tmp_repo="$(mktemp -d /tmp/zeppelin-repo-XXXXX)"
 
+  # build with scala-2.10
   echo "mvn clean install -Ppublish-distr \
     -Dmaven.repo.local=${tmp_repo} \
     ${PUBLISH_PROFILES} ${PROJECT_OPTIONS}"
   mvn clean install -Ppublish-distr -Dmaven.repo.local="${tmp_repo}" \
     ${PUBLISH_PROFILES} ${PROJECT_OPTIONS}
   if [[ $? -ne 0 ]]; then
-    echo "Build failed."
+    echo "Build with scala 2.10 failed."
+    exit 1
+  fi
+
+  # build with scala-2.11
+  "${BASEDIR}/change_scala_version.sh" 2.11
+
+  echo "mvn clean install -Ppublish-distr \
+    -Dmaven.repo.local=${tmp_repo} -Dscala-2.11 \
+    ${PUBLISH_PROFILES} ${PROJECT_OPTIONS}"
+  mvn clean install -Ppublish-distr -Dmaven.repo.local="${tmp_repo}" -Dscala-2.11\
```

Comment:
to be consistent with other places such as travis ci config, I would use p_scalal-2.11